guides/common/attributes-satellite.adoc

```diff
@@ -8,6 +8,10 @@
 :BaseURL: {RHDocsBaseURL}red_hat_satellite/{ProjectVersion}/html-single/
 
 // URLs (published on Red Hat Portal)
+// IMPORTANT: When you change any guide URLs, also update the following:
+// - guides/upstream_filename_to_satellite_link.json
+// - lib/foreman_theme_satellite/documentation.rb in RedHatSatellite/foreman_theme_satellite
```

Comment:
Actually you don't have to update this one right away. Once the new TOC will be generated, it will detect the incompatible links automatically. 
To fix the incompatibility a pr with the new toc and a fix to documentation.rb should be created.

Reply:
Ideally we would sync that mapping so we don't maintain the same thing in 3 places, but I explicitly asked for this because it tracks the related places. I don't think a docs writer monitors foreman_theme_satellite and may not be aware of it. After https://github.com/RedHatSatellite/foreman_theme_satellite/pull/58 it will have a mapping that must be maintained.